engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -250,6 +247,17 @@ paths:
           description: >
             Returns only workflows with the specified name.  If specified multiple times,
             returns workflows with any of the specified names.
+        - name: id
+          required: false
+          in: query
+          type: array
+          items:
+            type: string
+          collectionFormat: multi
+          pattern: ^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}$
```

Comment:
This pattern? No. Our implementation of `WorkflowId.fromString`? Maybe? I don't believe we have a spec internally. I also don't know if swagger-ui actually uses this swagger-spec pattern?